manifests/pki/certificate.pp

```diff
@@ -14,12 +14,13 @@
 
 
 define strongswan::pki::certificate (
-  $ca_name            = $::strongswan::pki::ca::ca_name,
-  $ca_certificate_dir = $::strongswan::pki::ca::certificate_dir,
-  $ca_private_key_dir = $::strongswan::pki::ca::private_key_dir,
-  $certificate_dir    = '/etc/strongswan/ipsec.d/certs',
-  $private_key_dir    = '/etc/strongswan/ipsec.d/private',
-  $common_name        = $::fqdn,
+  $ca_name            = $strongswan::pki::ca::ca_name,
+  $ca_certificate_dir = $strongswan::ca_certificate_dir,
+  $ca_private_key_dir = $strongswan::private_key_dir,
+  $certificate_dir    = $strongswan::certificate_dir,
+  $private_key_dir    = $strongswan::private_key_dir,
+  $strongswan_dir     = $strongswan::strongswan_dir,
+  $common_name        = $fqdn,
```

Comment:
This should be `facts['fqdn']` I think? `$::fqdn` Is a fact, accessed as top scope variable.

Reply:
Besides that everything looks good! I can merge it after you updated this part.